bin/minimize_vcf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3.8
+
+import pandas as pd
+import sys
+import io
+
+def main():
+    inputvcf = sys.stdin.read()
+    df = pd.read_csv(io.StringIO(inputvcf), sep="\t", comment="#", header=None)
+    df = df.rename(columns={
+        0: '#CHROM',
+        1: 'POS',
+        2: 'ID',
+        3: 'REF',
+        4: 'ALT',
+        5: 'QUAL',
+        6: 'FILTER',
+        7: 'INFO',
+        8: 'FORMAT',
+    })
+    df[["INFO"]] = "."
+    df = df[df.FILTER == "PASS"]
+    vcfbody = df.to_csv(sep="\t", index=False)
+
+    vcfformatheader = "\n".join(line for line in inputvcf.splitlines() if line.startswith("##FORMAT"))
```

Comment:
This looks fine, I want to double check it could result any errors without the FORMAT.

Reply:
yes it did